src-plugins/itkFilters/itkFiltersBinaryCloseProcess_p.h

```diff
@@ -72,8 +72,8 @@ class itkFiltersBinaryCloseProcessPrivate : public itkMorphologicalFiltersProces
         QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
 
         if (isRadiusInPixels)
-            newSeriesDescription += " Bin.Close filter\n("+ QString::number(radius[0])+", "+
-            QString::number(radius[1])+", "+ QString::number(radius[2])+" pixels)";
+            newSeriesDescription += " Bin.Close filter\n("+ QString::number(floor(radius[0]))+", "+
+            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
```

Comment:
(Sorry I see this has been fixed. Should have read more comments before responding...)